diff --git a/CMakeLists.txt b/CMakeLists.txt
index 93883600..29b5ceff 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,7 +19,7 @@ IF(APPLE)
     message(STATUS "CMAKE_OSX_ARCHITECTURES=${CMAKE_OSX_ARCHITECTURES}")
 ENDIF(APPLE)
 
-set(BIN_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
+set(BIN_OUTPUT_DIRECTORY "${CMAKE_INSTALL_BINDIR}")
 set(INSTALL_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/install")
 set(BUILD_SHARED_LIBS FALSE)
 
diff --git a/src/libcommonserver/CMakeLists.txt b/src/libcommonserver/CMakeLists.txt
index c258dbf7..729b783e 100644
--- a/src/libcommonserver/CMakeLists.txt
+++ b/src/libcommonserver/CMakeLists.txt
@@ -124,6 +124,6 @@ elseif(APPLE)
         utf8proc)
 else()
     target_link_libraries(${libcommonserver_NAME}
-        "/usr/local/lib/libxxhash.so"
+        xxHash::xxhash
         utf8proc)
 endif()

